Project.toml

```diff
@@ -21,7 +21,7 @@ LogExpFunctions = "0.3.2"
 Reexport = "1"
 Rmath = "0.4, 0.5, 0.6, 0.7"
 SpecialFunctions = "0.8, 0.9, 0.10, 1.0, 2"
-julia = "1"
+julia = "1.6"
```

Comment:
Unfortunately, we can't do this in a patch release, the registry doesn't allow it.

Reply:
Can you finish this PR?

Reply:
Waiting on feedback on the general issue: https://github.com/JuliaStats/StatsFuns.jl/issues/136#issuecomment-1085700322